config/value.go

```diff
@@ -387,12 +387,7 @@ func convertValue(value interface{}, targetType reflect.Type) (interface{}, erro
 
 // PopulateStruct fills in a struct from configuration
 func (cv Value) PopulateStruct(target interface{}) error {
-	if !cv.HasValue() {
-		return nil
-	}
-
 	_, err := cv.valueStruct(cv.key, target)
```

Comment:
Not sure if we need `cv.valueStruct()` method now since first parameter returned on valueStruct is never used.  We can just have this method signature for PopulateStruct on line 401 below that would just return `error`.